packages/OreIdProfile/src/ButtonOutline/ButtonOutline.tsx

```diff
@@ -2,22 +2,22 @@ import React from "react";
 
 import styles from "./ButtonOutline.module.scss";
 
-interface ButtonOutlineProps {
+interface ButtonOutlineProps extends React.HTMLProps<HTMLButtonElement> {
 	fontColor?: string;
-	label: string;
-	onClick: () => void;
+	type?: any
 }
 
 export const ButtonOutline: React.FC<ButtonOutlineProps> = ({
-	onClick,
-	label,
 	fontColor = "#fff",
+	children,
+	...rest
 }) => (
 	<button
 		className={styles.ButtonOutline}
 		style={{ color: fontColor, borderColor: fontColor }}
-		onClick={onClick}
+		{...rest}
```

Comment:
This way, you will create the prop `label={rest.label}`. It is better to separate the "additional" props on deconstruct.

Also putting `{...rest}` last, if you send `className` or `styles` it will overwrite those props. To prevent this, you should declare put `{...rest}` first, and the other props after.

Reply:
https://github.com/TeamAikon/oreid-react-components/pull/64/commits/d0b9707bf850f3c59103c68e7e79c13e549db06f